src/qibolab/platforms/platform.py

```diff
@@ -68,11 +68,11 @@ def set_lo_twpa_power(self, qubit, power):
     def get_lo_twpa_power(self, qubit):
         return self.qubits[qubit].twpa.local_oscillator.power
 
-    def set_attenuation(self, qubit, att):
-        raise_error(NotImplementedError, f"{self.name} does not support attenuation.")
+    def set_attenuation(self, qubit: Qubit, att):
+        self.qubits[qubit.name].attenuation = att
 
-    def get_attenuation(self, qubit):
-        raise_error(NotImplementedError, f"{self.name} does not support attenuation.")
+    def get_attenuation(self, qubit: Qubit):
+        return self.qubits[qubit.name].attenuation
```

Comment:
Here there is an asymmetry given that in qibocal we set the attenuation by passing the `Qubit` directly and not his id. I could change things in qibocal to make it compatible with the other methods.
This is also why tests are failing I believe.

Reply:
Then, in principle you could also do
```py
return qubit.attenuation
```
directly then, right? 

Personally, I think it looks cleaner if you pass the `Qubit` directly, then you don't need to do `self.qubits[qubit]`, but I think we should be consistent among all these methods. 

I would prefer even more to do `qubit.attenuation = ...` directly in qibocal instead of `platform.set_attenuation(qubit, att)` as then we would not need all these methods at all, but this will probably not work with the `MultiqubitPlatform`.

Other than the interface, I see a more practical issue here which is you are adding the `attenuation` in the `Channel` object but then you are trying to access it from a `Qubit` object. Does the `Qubit` have attenuation? Maybe you need something like `qubit.drive.attenuation` and a way to choose which attenuation you are controlling (drive or readout).

Reply:
Thanks for the review @stavros11.

> Personally, I think it looks cleaner if you pass the `Qubit` directly, then you don't need to do `self.qubits[qubit]`, but I think we should be consistent among all these methods.

I can change it also in qibocal, when I was implementing the new layout I was not sure about which convention we were following. I can change it.

 
> I would prefer even more to do `qubit.attenuation = ...` directly in qibocal instead of `platform.set_attenuation(qubit, att)` as then we would not need all these methods at all, but this will probably not work with the `MultiqubitPlatform`.
> Other than the interface, I see a more practical issue here which is you are adding the `attenuation` in the `Channel` object but then you are trying to access it from a `Qubit` object. Does the `Qubit` have attenuation?
> Other than the interface, I see a more practical issue here which is you are adding the `attenuation` in the `Channel` object but then you are trying to access it from a `Qubit` object. Does the `Qubit` have attenuation? Maybe you need something like `qubit.drive.attenuation` and a way to choose which attenuation you are controlling (drive or readout).

You are right. According to the [runcard](https://github.com/qiboteam/qibolab/blob/dummy_attenuation/src/qibolab/runcards/qw5q_gold_qblox.yml) there is attenuation value for both readout (qrm) and drive (qcm). I will try to implement something like `qubit.drive.attenuation` or `qubit.readout.attenuation`.
I will also like to drop the getters and setters in qibocal but I think that until we don't implement #419 I still need those to deal with `MultiqubitPlatform`.